source/UF/FunExt-Properties.lagda

```diff
@@ -1,4 +1,5 @@
 Martin Escardo, 19th May 2018.
+Evan Cavallo, 13th March 2025.
```

Comment:
The TypeTopology convention is that you write "Added by Evan Cavallo, 13th March 2025" just before your addition, and after the addition you write "End of addition."